src/aks-preview/HISTORY.rst

```diff
@@ -9,8 +9,9 @@ If there is no rush to release a new version, please just add a description of t
 
 To release a new version, please select a new version number (usually plus 1 to last patch version, X.Y.Z -> Major.Minor.Patch, more details in `\doc <https://semver.org/>`_), and then add a new section named as the new version number in this file, the content should include the new modifications and everything from the *Pending* section. Finally, update the `VERSION` variable in `setup.py` with this new version number.
 
-Pending
+2.0.0b1
 +++++++
+* Replace `guardrails` parameters with `safeguards`.
```

Comment:
Since this change is a breaking change, it is better to specify "breaking change" in the history notes, so that users can be easily aware of that.

Reply:
@yanzhudd, does it matter even if the feature was never released?

Reply:
The major version was also increased too, users can see that